VERSION_NUMBER

```diff
@@ -1 +1 @@
-1.7.0
+1.7.105
```

Comment:
Is this change intended? Does 105 have a special meaning?

Reply:
Yes, this is for the version number in test pypi. We have to use a different number each time we have a release candidate. Once the final verification is complete, we will change it back to 1.7.0 in the release branch.

Reply:
There is no special meaning other than I started from 101 and increment by 1 for each release candidate.